server/events/vcs/github_client.go

```diff
@@ -113,6 +139,60 @@ func (g *GithubClient) CreateComment(repo models.Repo, pullNum int, comment stri
 	return nil
 }
 
+func (g *GithubClient) HideOldComments(repo models.Repo, pullNum int) error {
+	var allComments []*github.IssueComment
+	for page := 0; ; {
+		comments, resp, err := g.client.Issues.ListComments(g.ctx, repo.Owner, repo.Name, pullNum, &github.IssueListCommentsOptions{
+			Sort:        "created",
+			Direction:   "asc",
+			ListOptions: github.ListOptions{Page: page},
+		})
+		if err != nil {
+			return err
+		}
+		allComments = append(allComments, comments...)
+		if resp.NextPage == 0 {
+			break
+		}
+		page = resp.NextPage
+	}
+
+	for _, comment := range allComments {
+		if comment.User != nil && comment.User.GetLogin() != g.user {
```

Comment:
Was testing this out but running into issues with case sensitive username.
``suggestion
		if comment.User != nil && !strings.EqualFold(comment.User.GetLogin(), g.user)
```

Reply:
Okay I'll add it.